getting-started/templates/OnPrem/storage-values.yaml

```diff
@@ -11,14 +11,9 @@
 ## be overridden with custom values if required.
 
 ## Hostname of an external S3 provider.
-# <ATTENTION> To connect to an external S3 provider, set s3Host, s3Scheme, s3Port and s3Region.
+# <ATTENTION> To connect to an S3 provider, set s3Host, s3Scheme, s3Port and s3Region.
```

Comment:
Suggestion: "To connect to an S3 provider, set the values of the following variables: s3Host, s3Scheme, s3Port, and s3Region."